jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -98,38 +95,21 @@ def process(self, input, *args, **kwargs):
         if not (result.meta.exposure.type.lower() in IMAGING_TYPES and self.sip_approx):
             return result
 
+        (result, )
```

Comment:
this is not being assigned to a variable or anything, should this line be deleted?

Reply:
yes... some crumbs.

Reply:
fixed